### PR TITLE
ConstantFolding for Ceiling and Floor

### PR DESCRIPTION
**Pull request by aprocter**
~Merge after #3319~ ~This has been merged, so okay to go ahead.~ ~Merge after #3328 :)~

### CODE REVIEW

File: src/ngraph/runtime/cpu/cpu_builder.cpp [hunk 1]
```diff
@@ -470,6 +470,18 @@ namespace ngraph
                 BUILD_UNARY_ELEMWISE_CF_FUNCTOR(runtime::cpu::kernel::sqrt);
             }
 
+            template <>
+            NodeExecutorTy Builder::BUILDER_CF_DECL(ngraph::op::Floor)
```
Comment (nmostafa) [inline src/ngraph/runtime/cpu/cpu_builder.cpp]: No unit-tests ?

Comment (aprocter) [reply src/ngraph/runtime/cpu/cpu_builder.cpp]: @nmostafa I didn't realize until you nudged me on this that you guys _do_ have tests for these functors. I thought they were just going untested. I'll add some tests for the new functors I have been adding. Please switch to `changes requested`, or I can follow on in a later PR.

**Comment by aprocter** [comment]
Merge after #3328

File: src/ngraph/runtime/cpu/cpu_builder.cpp (hunk 1 above)
Comment (aprocter) [reply src/ngraph/runtime/cpu/cpu_builder.cpp]: Have added some tests.

**Pull request merged**